File: src/test/resources/expectedfolder/admin/Message.cs
using System;
using org.codejuicer.java2csharp.testclasses;

namespace org.codejuicer.java2csharp.testclasses.admin
{
	public abstract class Message
	{
		private Header header;

		protected internal Message()
			: base()
		{
		}

		public Message(Header header)
			: base()
		{
			this.header = header;
		}

		public virtual Header Header
		{
			get
			{
				return header;
			}
			set
			{
				Header header = value;
				this.header = header;
			}
		}
	}
}

File: src/test/java/org/codejuicer/java2csharp/testclasses/admin/Message.java
/*
 * Copyright 2014 Giuseppe Gerla. All Rights Reserved.
 *
 * Licensed under the GNU General Public License, Version 2.0;
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.gnu.org/licenses/gpl2.txt
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

package org.codejuicer.java2csharp.testclasses.admin;

import org.codejuicer.java2csharp.testclasses.Header;

public abstract class Message {
	private Header header;

	protected Message() {
		super();
	}

	public Message(Header header) {
		super();
		this.header = header;
	}

	public Header getHeader() {
		return header;
	}

	public void setHeader(Header header) {
		this.header = header;
	}
}